package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lana/b2c-mapp-ui",
-  "version": "3.2.2",
+  "version": "4.0.0",
```

Comment:
We can keep this release just version `3.2.3`.

Reply:
Keeping 4.0.0... This is not a patch version that it implies a breaking change adding a component it was not present before.

Reply:
I was just thinking it may create a bit of instability in versions.